sdk/cosmos/azure-cosmos-spark_3-0_2-12/src/main/scala/com/azure/cosmos/SparkBridgeInternal.scala

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos
+
+import com.azure.cosmos.models.FeedRange
+
+private[cosmos] object SparkBridgeInternal {
+  def trySplitFeedRange
+  (
+    container: CosmosAsyncContainer,
+    feedRange: String,
+    targetedCountAfterSplit: Int
+  ): Array[String] = {
+
+    val list = container
+      .trySplitFeedRange(FeedRange.fromString(feedRange), targetedCountAfterSplit)
+      .block
+
+    val array = new Array[String](list.size)
+    for (i <- 0 to list.size - 1) {
```

Comment:
Hmm - I prefer the explicit version - because folks who don't work with Scala all the time are not forced to lookup whether until keyword is inclusive or exclusive. But can live with both versions - will change it in next iteration.